### PR TITLE
summarise() regrouping information, take 2

### PR DESCRIPTION
**Pull request by romainfrancois**

### CODE REVIEW

File: tests/testthat/test-summarise-errors.txt [hunk 1]
```diff
@@ -4,24 +4,50 @@ Messages about .groups=
...
 
 # A tibble: 1 x 2
 # Groups:   x, y [1]
       x     y
   <dbl> <dbl>
 1     1     2
 
+> eval_bare(expr(tibble(x = 1, y = 2) %>% rowwise() %>% summarise()), env(
++   global_env()))
+Message: `summarise()` grouping output by '' (override with `.groups` argument)
```
Comment (romainfrancois) [inline tests/testthat/test-summarise-errors.txt:47]: That looks weird

Comment (hadley) [reply tests/testthat/test-summarise-errors.txt:47]: Ooops, that should be the same as `df %>% group_by(x) %>% summarise()`, I think?

Comment (romainfrancois) [reply tests/testthat/test-summarise-errors.txt:47]:
``` r
library(dplyr, warn.conflicts = FALSE)
library(rlang)

local({
  tibble(x = 1, y = 2) %>% group_by(x, y) %>% summarise()
  tibble(x = 1, y = 2) %>% group_by(x, y) %>% summarise(z = c(2,2))
  tibble(x = 1, y = 2) %>% group_by(x) %>% summarise()
  tibble(x = 1, y = 2) %>% rowwise(x, y) %>% summarise()
  tibble(x = 1, y = 2) %>% rowwise() %>% summarise()
})
#> `summarise()` regrouping output by 'x' (override with `.groups` argument)
#> `summarise()` regrouping output by 'x', 'y' (override with `.groups` argument)
#> `summarise()` ungrouping output (override with `.groups` argument)
#> `summarise()` grouping output by 'x', 'y' (override with `.groups` argument)
#> `summarise()` grouping output by '' (override with `.groups` argument)
#> # A tibble: 1 x 0
```

<sup>Created on 2020-05-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Comment (romainfrancois) [reply tests/testthat/test-summarise-errors.txt:18]: I finally found that `verify_output()` has an `env=` argument so `test-summarise-errors.txt` should be more readable. How about : 

```
> ignored <- tibble(x = 1, y = 2) %>% rowwise(x, y) %>% summarise()
Message: `summarise()` regrouping output by rows: 'x', 'y' (override with `.groups` argument)

> ignored <- tibble(x = 1, y = 2) %>% rowwise() %>% summarise()
Message: `summarise()` regrouping output by rows (override with `.groups` argument)
```

Comment (hadley) [reply tests/testthat/test-summarise-errors.txt]: But they're no longer being grouped by rows

Comment (romainfrancois) [reply tests/testthat/test-summarise-errors.txt]: oh ok, got it now :-)

**Pull request merged**